modules/gapi/src/backends/fluid/gfluidimgproc.cpp

```diff
@@ -1980,11 +1990,38 @@ struct Mapper {
         return u;
     }
 };
+}  // namespace linear
+
+namespace linear32f {
+struct Mapper {
+    typedef float alpha_type;
+    typedef int   index_type;
+    constexpr static const float unity = 1;
+
+    typedef MapperUnit<float, int> Unit;
+
+    static inline Unit map(double ratio, int start, int max, int outCoord) {
+        float f = static_cast<float>((outCoord + 0.5) * ratio - 0.5);
+        int s = cvFloor(f);
+        f -= s;
+
+        Unit u;
+
+        u.index0 = std::max(s - start, 0);
+        u.index1 = ((f == 0.0) || s + 1 >= max) ? s - start : s - start + 1;
```

Comment:
how is  correct to compare f with 0.0 without epsilon?

Reply:
could you describe such calculations in `/* comment */` please - what is it?

Reply:
> could you describe such calculations in `/* comment */` please - what is it?

This is calculation of the map of Resize F32C1. This can be understood from the name of the structure in which this method is located and from the name of the method itself. Additional comments are superfluous in my opinion.